FreeIT/src/lesson3/Task1.java

```diff
@@ -0,0 +1,19 @@
+package lesson3;
+
+//1)Начав тренировки, спортсмен в первый день пробежал 10 км. Каждыйдень он увеличивал дневную норму на 10% нормы 
+//предыдущего дня. Какойсуммарный путь пробежит спортсмен за 7 дней?
+
+public class Task1 {
+
+	public static void main(String[] args) {
+		float dayProbeg = 10f;
```

Comment:
Try not to use russian words. dayDistance will be better

Reply:
Ok